values/gitea/gitea.gotmpl

```diff
@@ -28,6 +28,19 @@ podDns:
 
 resources: {{- $g.resources.gitea | toYaml | nindent 2 }}
 
+affinity:
+  podAntiAffinity:
```

Comment:
Unfortunately this fails in the upgrade scenario, since updates are not allowed to a running stateful set.